### PR TITLE
[Fluent] Tilelayout

### PR DESCRIPTION
**Pull request by magdalenaan**
related https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/tilelayout/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@use "../core" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-tilelayout--layout() {
+
+    .k-tilelayout {
+        border-width: var( --kendo-tilelayout-border-width, $kendo-tilelayout-border-width );
```
Comment (TeyaVes) [inline packages/fluent/scss/tilelayout/_layout.scss:7]: the fallback should be with an interpolation: #{$kendo-tilelayout-border-width}

the same goes for all vars

File: packages/fluent/scss/tilelayout/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    .k-tilelayout-item.k-state-focused,
+    .k-tilelayout-item.k-focus {
+        @include box-shadow($kendo-tilelayout-card-focus-shadow);
+    }
+
+    .k-layout-item-hint {
+        @include fill(
+            $bg: $kendo-tilelayout-hint-bg,
+            $border: $kendo-tilelayout-hint-border
+        );
```
Comment (TeyaVes) [inline packages/fluent/scss/tilelayout/_theme.scss:24]: use CSS vars

File: packages/fluent/scss/tilelayout/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+$kendo-tilelayout-card-border-width: $kendo-card-border-width !default;
+$kendo-tilelayout-card-focus-shadow: $kendo-card-focus-shadow !default;
+
+$kendo-tilelayout-hint-border-width: 1px !default;
+$kendo-tilelayout-hint-border-radius: $kendo-border-radius-lg !default;
+
+$kendo-tilelayout-bg: get-theme-color-var( neutral-10 ) !default;
+
+$kendo-tilelayout-hint-bg: rgba(255, 255, 255, .2) !default;
+$kendo-tilelayout-hint-border: $kendo-component-border !default;
```
Comment (TeyaVes) [inline packages/fluent/scss/tilelayout/_variables.scss:17]: use CSS var as it's coming from the core file

**Review by TeyaVes — CHANGES_REQUESTED** [review]

File: packages/fluent/scss/tilelayout/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+@use "../core/" as *;
+@use "../core/color-system" as *;
+@use "../card/_variables.scss" as *;
+@use "sass:color";
+
+// TileLayout
+$kendo-tilelayout-border-width: 0px !default;
+$kendo-tilelayout-card-border-width: $kendo-card-border-width !default;
```
[the remark this answers is not in the document]
Comment (magdalenaan) [reply packages/fluent/scss/tilelayout/_variables.scss]: The design is correct so there is a shadow instead of a border.

**Review by TeyaVes — APPROVED** [review]

**Pull request merged**